core/src/test/scala/org/apache/spark/SparkConfSuite.scala

```diff
@@ -354,6 +354,16 @@ class SparkConfSuite extends SparkFunSuite with LocalSparkContext with ResetSyst
     }
   }
 
+  test("SPARK-27244 toDebugString should redact passwords") {
+    val EXPECTED_LOG = s"dummy.password=${Utils.REDACTION_REPLACEMENT_TEXT}"
+    val conf = new SparkConf().set("dummy.password", "dummy-password")
+    conf.validateSettings()
+
+    val debugString = conf.toDebugString
```

Comment:
The `debugString` variable is not needed and it can be inlined.

Reply:
Is this the reason the build is failing? I will do the changes

Reply:
No. The first failure was something with infrastructure as it took less than 7 minutes but the second one is running more than 3 hours which is normal.

Reply:
Ok thanks, I will make that variable inline